### DISCUSSION TITLE
Welcome to boxbilling Discussions!

### DISCUSSION THREAD

**Discussion by fordnox** [boxbilling/boxbilling#793] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by timothygwebb** [comment]
Greetings fellow community,

My name is Timothy and I am a collaborator for boxbilling. I currently work as a web hosting provider and consultant. I am glad to see that I was able to get boxbilling back in the green again with the builds. I will be working with the community making sure that boxbilling is php8 compatible. 👍🏻

**Comment by ShadowCrownServices** [comment]
Hi, @fordnox 
I'm new to box billing and have not long installed it on my server...
I have noticed it is packaged with "Selling SolusVM VPS" and one or two references to Plesk via here (Github) and google search!

If it is possible to connect with Plesk, is there an easy-to-follow work-through to achieve this, please?

Kind Regards
John
